src/main/drivers/accgyro/accgyro_spi_icm426xx.c

```diff
@@ -188,13 +203,12 @@ void icm426xxGyroInit(gyroDev_t *gyro)
 
     uint8_t odrIdx = 0;
     bool supportedODRFound = false;
-
     if (gyro->gyroRateKHz) {
         const uint8_t gyroSyncDenominator = gyro->mpuDividerDrops + 1; // rebuild it in here, see gyro_sync.c
         const uint8_t desiredODR = 8 / gyroSyncDenominator;
-        for (unsigned i = 0; i < ARRAYLEN(odrConfigLUT); i++) {
-            if (odrConfigLUT[i].odr == desiredODR) {
-                odrIdx = odrConfigLUT[i].odr_idx;
+        for (unsigned i = 0; i < ARRAYLEN(odrLUT); i++) {
```

Comment:
Maybe: Store gyroSyncDenominator (or gyro->mpuDividerDrops, or even gyroRateKHz) in lookup table directly. With gyroRateKHz and nearest search, actual configured value may be stored back into gyroRateKHz (instead forcing 1kHz)

Reply:
This part is kind of deprecated because we always use a denominator of 1. But I think this is a good idea. I don't dare to touch this right here in this PR here because I don't want to risk introducing a new bug while this PR is meant to just fix the AAF configuration.

I'll make a quick follow up refactoring PR for that.

Reply:
Ok, changed my mind. I simplified this part and also got a bunch of flash savings as a bonus.